<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWindow</class>
 <widget class="QMainWindow" name="adminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="Pages">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_0">
       <widget class="QLineEdit" name="usernameLineEdit">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>160</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>ADMIN LOGIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="usernameLabel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>120</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
       <widget class="QLabel" name="passwordLabel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>160</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>190</y>
          <width>168</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="loginButton">
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelLoginButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_1">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>190</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>admin login stuff</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="modifyStadiumPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QTableView" name="stadiumTableView"/>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEditTeamName"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStadiumName"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditSeatingCap"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditLocation"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditConference"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditSurfaceType"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditRoofType"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStarPleyer"/>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAddStadium">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
